.pre-commit-config.yaml

```diff
@@ -48,7 +48,7 @@ repos:
     rev: 6.0.0
     hooks:
     -   id: flake8
-        args: ["--exclude", ".*,tests/eval_files/*"]
+        types: ["python", "pyi"]
```

Comment:
I .. got incredibly confused and thought flake8 was on it's way to including `.pyi` by default (see https://github.com/PyCQA/flake8/issues/1806 :no_mouth:) and went ahead and included it; but even after realizing it's not meant to handle `.pyi` by default I realized there weren't that many errors in there anyway - so not much reason to have a separate environment, can just handle that with per-file-ignores.